parquet/src/test/java/org/apache/iceberg/parquet/TestDictionaryRowGroupFilter.java

```diff
@@ -911,20 +928,31 @@ public void testTypePromotion() {
 
   @Test
   public void testFixedLenByteArray() {
-    // This test is to validate the handling of FIXED_LEN_BYTE_ARRAY Parquet type being dictionary encoded.
+    // This test is to validate the handling of FIXED_LEN_BYTE_ARRAY Parquet type
+    // being dictionary encoded.
     // (No need to validate all the possible predicates)
 
-    Assert.assertTrue("decimal_fixed should be dictionary encoded",
-        getColumnForName(rowGroupMetadata, "_decimal_fixed").getEncodings().contains(Encoding.RLE_DICTIONARY));
-
-    boolean shouldRead = new ParquetDictionaryRowGroupFilter(SCHEMA,
-        greaterThanOrEqual("decimal_fixed", BigDecimal.ZERO)).shouldRead(parquetSchema, rowGroupMetadata,
-            dictionaryStore);
-    Assert.assertTrue("Should read: Half of the decimal_fixed values are greater than 0", shouldRead);
-
-    shouldRead = new ParquetDictionaryRowGroupFilter(SCHEMA, lessThan("decimal_fixed", DECIMAL_MIN_VALUE))
-        .shouldRead(parquetSchema, rowGroupMetadata, dictionaryStore);
-    Assert.assertFalse("Should not read: No decimal_fixed values less than -1234567890.0987654321", shouldRead);
+    switch (writerVersion) {
+      case PARQUET_1_0:
```

Comment:
Normally, we would use `Assume` to simply skip this test for Parquet v1. Would you mind doing that here? I don't think we need to validate that Parquet doesn't dictionary encode.